log/value.go

```diff
@@ -39,6 +39,36 @@ func Timestamp(t func() time.Time) Valuer {
 	return func() interface{} { return t() }
 }
 
+// TimestampFormat returns a Valuer that produces a TimeFormat value from
+// layout and the time returned by t. Users will probably want to use
+// DefaultTimestamp or DefaultTimestampUTC.
+func TimestampFormat(t func() time.Time, layout string) Valuer {
```

Comment:
I'm going to stick with the name `t` because the returned time doesn't _have_ to represent the current time. I agree that the comment will need to be reworded when `TimeFormat` is not exported. Thanks for the wordsmithing.